cmd/srcd-server/engine/sql.go

```diff
@@ -36,7 +36,7 @@ func (s *Server) SQL(req *api.SQLRequest, stream api.Engine_SQLServer) error {
 		Net:                  "tcp",
 		Addr:                 gitbase.Name,
 		AllowNativePasswords: true,
-		MaxAllowedPacket:     32 * (2 << 10),
+		MaxAllowedPacket:     32 * (1 << 20),
```

Comment:
Thanks for noticing! I suggest to add a comment with the intended value, because the field name does not tell if it's in bytes, bits, MiB... 
```suggestion
		MaxAllowedPacket:     32 << 20, // 32 MiB
```